utils/package_diff_utils.go

```diff
@@ -48,61 +47,29 @@ type PackageInfo struct {
 	Size    string
 }
 
-func contains(info1 []PackageInfo, keys1 []string, key string, value PackageInfo) (int, bool) {
-	if len(info1) != len(keys1) {
-		return 0, false
-	}
-	for i, currVal := range info1 {
-		if !reflect.DeepEqual(currVal, value) {
-			continue
-		}
-		// Check if both global or local installations by trimming img-id and layer hash
-		tempPath1 := strings.SplitN(key, "/", 3)
-		tempPath2 := strings.SplitN(keys1[i], "/", 3)
-		if len(tempPath1) != 3 || len(tempPath2) != 3 {
-			continue
-		}
-		if tempPath1[2] == tempPath2[2] {
-			return i, true
-		}
-	}
-	return 0, false
-}
 
 func multiVersionDiff(infoDiff []MultiVersionInfo, key string, map1, map2 map[string]PackageInfo) []MultiVersionInfo {
```

Comment:
maybe more descriptive vars below besides "key" and "value" types would be helpful to understanding what's going on here